chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java

```diff
@@ -349,7 +349,7 @@
      * Date the listing's Real World Testing Plan was submitted
      */
     @XmlElement(name = "rwt_plan_submission_date", nillable = true, required = false)
-    private Date rwtPlanSubmissionDate;
+    private Long rwtPlanSubmissionDate;
```

Comment:
Is this documented in the "date" page on the wiki? I'm not sure that the RWT submission Date shouldn't be a "Date" type

Reply:
A Java Date represents a timestamp and we usually send that back as a long. If this is supposed to be a date-without-time type of date then I think it should be a different type entirely. I believe LocalDate? That would match the data model type.